tests/integration/plugins/test_go.py

```diff
@@ -124,6 +126,10 @@ def test_go_generate(new_dir, partitions):
     assert output == "This is a generated line\n"
 
 
+@pytest.mark.skipif(
```

Comment:
This is solvable in the makefile, but it's become a big enough hassle that it deserves its own ticket.